twiml/voice/refer/refer-4/refer-4.3.x.js

```diff
@@ -2,9 +2,9 @@ const VoiceResponse = require('twilio').twiml.VoiceResponse;
 
 const response = new VoiceResponse();
 const refer = response.refer({
-    'action': '/handleRefer',
-    'method': 'POST'
+    action: '/handleRefer',
```

Comment:
Interesting...did you change this or did the formatting change?

Reply:
This as an automated formatting change